src/auth.c

```diff
@@ -1716,13 +1716,16 @@ int pr_auth_chroot(const char *path) {
 
   tz = pr_env_get(session.pool, "TZ"); 
   if (tz == NULL) {
+    char tzbuf[32];
+    long off = (timezone / 60 / 60) - daylight;
+    snprintf(tzbuf, sizeof(tzbuf), "%s%ld", daylight ? tzname[1] : tzname[0], off);
```

Comment:
If you update this PR to use ":/etc/localtime", and maybe throw in a call to tzset(3) following the setting of the TZ environment variable, I'll merge it, and we'll go from there.  Sound good?